packages/widget-v2/src/widget/wallet/CosmosProvider.tsx

```diff
@@ -13,9 +13,6 @@ export const CosmosProvider: React.FC<CosmosProviderProps> = ({
     <GrazProvider grazOptions={{
       chains: mainnetChains,
       autoReconnect: true,
-      onNotFound: () => {
-        console.warn("wallet not found")
-      },
```

Comment:
I think we should keep this around, maybe we can handle it better in the future and but this way we won't forget this case can happen